python/paddle/tensor/linalg.py

```diff
@@ -5038,3 +5038,91 @@ def __check_ranges(D, ranges):
         hist /= s
 
     return (hist, edges)
+
+
+def ormqr(input, tau, other, left=True, transpose=False):
```

Comment:
The parameter naming does not comply with regulations, please refer to the official website documentation of [API design guidelines standard](https://www.paddlepaddle.org.cn/documentation/docs/zh/dev_guides/api_contributing_guides/api_design_guidelines_standard_cn.html)

Reply:
Ok, I will be re-pushing the revised code tomorrow